remoteit/desktop
# [DESK-727] ssh copy username

### Description



### Test plan



1. Sign in
2. Go to one device
3. Connect to a service SSH
4. Set up the username
5. Copy the command to connect
6. You should see the username on the clipboard

### Ticket

https://remoteit.atlassian.net/browse/DESK-727
